Ошибка при использовании сочетания клавиш Ctrl+Enter

Внутри редактора Code-Basics поддерживается сочетание клавиш Ctrl+Enter, однако при его использовании проверка решения заканчивается примерно такой ошибкой:

```
/usr/lib/node_modules/@github/query-selector/dist/index.umd.js:93
    throw new QueryError("Element not found: <".concat(klass.name, "> ").concat(selectors));
    ^

QueryError: Element not found: <HTMLElement> p
    at query (/usr/lib/node_modules/@github/query-selector/dist/index.umd.js:93:11)
    at /exercises-css/modules/10-basics/40-selectors-by-class/test.js:4:19
    at test (/exercises-css/src/tests.js:14:5)
    at Object.<anonymous> (/exercises-css/modules/10-basics/40-selectors-by-class/test.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  framesToPop: 1
}
make: *** [Makefile:2: test] Error 1
#<Process::Status: pid 11680 exit 2>
```

При нажатии мышкой на кнопку `Проверить` все ОК.

Проверял на уроках 3-7 в курсе по CSS.

--- a/app/packs/src/lesson/components/Editor.jsx
+++ b/app/packs/src/lesson/components/Editor.jsx
@@ -92,7 +92,6 @@ const Editor = () => {
   const handleRunCheck = (e) => {
     const value = e.getValue();
     dispatch(actions.runCheck({ lessonVersion, editor: { content: value } }));
-    debugger;
   };
 
   const options = {

--- a/app/packs/src/lesson/components/Editor.jsx
+++ b/app/packs/src/lesson/components/Editor.jsx
@@ -89,6 +89,12 @@ const Editor = () => {
     cm.replaceSelection(spaces);
   };
 
+  const handleRunCheck = (e) => {
+    const value = e.getValue();
+    dispatch(actions.runCheck({ lessonVersion, editor: { content: value } }));
+    debugger;
+  };
+
   const options = {
     autofocus: true,
     ...commonOptions,
@@ -96,9 +102,7 @@ const Editor = () => {
     indentUnit: getTabSize(language),
     extraKeys: {
       Tab: replaceTab,
-      'Ctrl-Enter': () => {
-        dispatch(actions.runCheck({ lessonVersion, editor: content }));
-      },
+      'Ctrl-Enter': handleRunCheck,
     },
   };
 

